### PR TITLE
:sparkles: feat: Mocked wallet in Cypress

### PR DESCRIPTION
**Pull request by localtechboy**
## Motivation and context

EthereumWalletMock for Cypress

## Does it fix any issue?

https://linear.app/synpress/issue/SYN-115/move-playwright-mocked-wallet-api-to-cypress

## Quality checklist

- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough e2e tests.

**⚠️👆 Delete any section you see irrelevant before submitting the pull request 👆⚠️**

### CODE REVIEW

File: wallets/ethereum-wallet-mock/src/cypress/support/mockEthereum.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+before(() => {
+  cy.visit('/', {
+    onBeforeLoad: (window) => {
+      window.Web3Mock.mock({
+        blockchain: 'ethereum',
+        wallet: 'metamask'
```
Comment (drptbl) [inline wallets/ethereum-wallet-mock/src/cypress/support/mockEthereum.ts:6]: should we add ability to specify mocked wallet here?

File: wallets/ethereum-wallet-mock/src/cypress/support/e2e.ts [hunk 2]
```diff
@@ -15,3 +15,9 @@
 
 // Import commands.js using ES2015 syntax:
 import './commands'
+import './mockEthereum'
+
+Cypress.on('uncaught:exception', () => {
+  // failing the test
+  return false
```
Comment (drptbl) [inline wallets/ethereum-wallet-mock/src/cypress/support/e2e.ts:22]: not sure if this is not going to fail a test when any critical error is thrown in browser console on any browser (not only related to wallet itself). do we want this behavior?

File: wallets/ethereum-wallet-mock/src/cypress/utils/installSynpress.ts [hunk 3]
```diff
@@ -21,7 +20,8 @@ export default function installSynpress(on: Cypress.PluginEvents, config: Cypres
     await initEthereumWalletMock(port)
   })
 
-  setupTasks(on)
+  // TODO: Implement tasks if needed
+  // setupTasks(on);
```
Comment (drptbl) [inline wallets/ethereum-wallet-mock/src/cypress/utils/installSynpress.ts]: do we want it here? we already import commands on `e2e.ts`

**Review by drptbl — CHANGES_REQUESTED** [review]
Great work sir, just need some answers to above - and changes if needed.

File: wallets/ethereum-wallet-mock/src/cypress/support/e2e.ts (hunk 2 above)
Comment (localtechboy) [reply wallets/ethereum-wallet-mock/src/cypress/support/e2e.ts:22]: It the opposite - it should prevent failing if possible.

File: wallets/ethereum-wallet-mock/src/cypress/support/mockEthereum.ts (hunk 1 above)
Comment (localtechboy) [reply wallets/ethereum-wallet-mock/src/cypress/support/mockEthereum.ts:6]: It's a wallet for initial mock. User is able to connect a different wallet using `connectToDapp`.

**Review by drptbl — APPROVED** [review]

**Pull request merged**